config/configmap/inferenceservice.yaml

```diff
@@ -140,7 +140,8 @@ data:
         "memoryRequest": "100Mi",
         "memoryLimit": "1Gi",
         "cpuRequest": "100m",
-        "cpuLimit": "1"
+        "cpuLimit": "1",
+        "storageSpecSecret": "storage-config"
```

Comment:
Secrets are namespace scope on Kubernetes. In KFP, they have a profile controller that replicates all the necessary namespace scoped resource when creating a new user. We can leverage the same capability to replicate the common secret for Kubeflow users. For standalone users, we are expecting them to create their own common secrets for their namespaces because we cannot authorize those users.
https://github.com/kubeflow/manifests/blob/master/apps/pipeline/upstream/base/installs/multi-user/pipelines-profile-controller/sync.py